types/googlemaps/tslint.json

```diff
@@ -1,79 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "adjacent-overload-signatures": false,
-        "array-type": false,
-        "arrow-return-shorthand": false,
-        "ban-types": false,
-        "callable-types": false,
-        "comment-format": false,
-        "dt-header": false,
         "npm-naming": false,
```

Comment:
How to correct this rule?

Reply:
Once you remove this rule, the error will show you how to fix it. In essence, you want to match the name of the package to match the npm package if this is for npm package. Otherwise add header with first line saying
`// Type definitions for non-npm package packagename`

Reply:
@sheetalkamat, do you suggest renaming a package?

<img src="https://user-images.githubusercontent.com/10235949/58726022-50ef7600-83e9-11e9-8999-e54126cbdf28.png" width="605" height="203" />